cmd/goal/account.go

```diff
@@ -579,6 +610,23 @@ var addParticipationKeyCmd = &cobra.Command{
 	},
 }
 
+var installParticipationKeyCmd = &cobra.Command{
+	Use:   "installpartkey",
+	Short: "Install a participation key",
+	Long:  `Install a participation key`,
```

Comment:
We should indicate that this does not change online status.

Reply:
Indeed, thanks.